client/components/category-pill-navigation/index.tsx

```diff
@@ -55,8 +55,20 @@ export const CategoryPillNavigation = ( {
 	};
 
 	useEffect( () => {
+		if ( ! listRef.current ) {
+			return;
+		}
+
 		checkScrollArrows();
-	}, [] );
+
+		const target = listRef.current.querySelector( '.is-active' );
+
+		target?.scrollIntoView( {
```

Comment:
Awesome idea, especially for cases when the page was opened with a category which is outside of the view 👍